File: Java/Math/Prime Number.java
//https://www.geeksforgeeks.org/problems/prime-number2314/1
class Solution {
    static boolean isPrime(int n) {
        // code here
        if (n <= 1) {
        return false;
    }
    
    for (int i = 2; i < n; i++) {
        if (n % i == 0) {
            return false;
        }
    }
    return true;
    }
}

File: Python/Math/Prime Number.py
//https://www.geeksforgeeks.org/problems/prime-number2314/1
class Solution:
    def isPrime(self, n):
        # code here
        if n <= 1:
            return False
        for i in range(2, int(n**0.5) + 1):
            if n % i == 0:
                return False
        return True
